server/handlers/tagHandler.js

```diff
@@ -8,13 +8,13 @@ const {
 const tagGetQuerySchema = Joi.object({
   limit: Joi.number().integer().greater(0).less(101),
   offset: Joi.number().integer().greater(-1),
-  owner_id: Joi.string(),
+  owner_id: Joi.string().default(null),
```

Comment:
If we default to null I think that will just return only tags with owner_id as null

Reply:
Ok I changed it to just .allow(null) per our conversation